src/preprocessing/passes/bv_to_int.cpp

```diff
@@ -139,13 +139,13 @@ Node BVToInt::makeBinary(Node n)
         // current has children, but we do not binarize it
         NodeBuilder<> builder(k);
         if (current.getKind() == kind::BITVECTOR_EXTRACT
-            || current.getKind() == kind::APPLY_UF)
+            || current.getKind() == kind::APPLY_UF || current.getKind() == kind::INT_TO_BITVECTOR)
```

Comment:
The robust check here is `current.getMetaKind() == metakind::PARAMETERIZED`. This indicates if the operator should be added when constructing the node.  Do update this.